src/main/java/core/basesyntax/Calculate.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Calculate {
+    private static final String SUPPLY = "supply,";
+    private static final String TO_BUY = "buy,";
+    private static final String RESULT = "result,";
+
+    public String calculateBuySupplyData(String data) {
```

Comment:
due to my comment regarding the class name, this method can be renamed to:
` public String calculateReportFrom(String data)`